Vokal-ObjC.xctemplate/___VARIABLE_classPrefix___CommonInitializedView.h

```diff
@@ -0,0 +1,38 @@
+//
+//  ___FILENAME___
+//  ___PACKAGENAME___
+//
+//  Created by ___FULLUSERNAME___ on ___DATE___.
+//___COPYRIGHT___
+//
+
+@import Foundation;
+
+@protocol ___VARIABLE_classPrefix___CommonInitializedView <NSObject>
+
+@property (readonly) BOOL oneTimeThingsAreSetUp;
```

Comment:
`(nonatomic, ...)` here and for others? Also for the classes that are conforming, worthwhile to be more explicit with `readwrite`?